### PR TITLE
[UBS courier] 'Номер замовлення' field is active when 'Hi' radio button is selected instead of be inactive #2599

### PR DESCRIPTION
**Pull request by SofiiaBaziv**
**Before**
'Номер замовлення' field is active when 'Hi' radio button is selected

**After**
'Номер замовлення' field is inactive when 'Hi' radio button is selected

### CODE REVIEW

**Review by idzhersh — APPROVED** [review]

**Review by AlekOK — APPROVED** [review]

File: src/app/component/ubs/components/ubs-order-details/ubs-order-details.component.ts [hunk 1]
```diff
@@ -148,6 +152,12 @@ export class UBSOrderDetailsComponent implements OnInit, OnDestroy {
     }
   }
 
+  clearOrderValues(): void {
+    this.additionalOrders.controls.forEach(element => {
```
Comment (JimHolman1199) [inline src/app/component/ubs/components/ubs-order-details/ubs-order-details.component.ts:156]: use map instead, https://lorenstewart.me/2017/01/22/javascript-array-methods-mutating-vs-non-mutating/

File: src/app/component/ubs/components/ubs-order-details/ubs-order-details.component.html [hunk 2]
```diff
@@ -166,7 +166,7 @@ <h5>Чи чекаєте Ви замовлення з нашого Еко-маг
         <div formArrayName="additionalOrders" *ngFor="let order of additionalOrders.controls; let i=index"
           class="form-group">
           <label class="label m-0">Введіть номер замовлення</label>
-          <input [formControlName]="i" class="shadow-none form-control border-input p-2" type="text" placeholder="xxxxxxxxxx">
+          <input [formControlName]="i" class="shadow-none form-control border-input p-2" type="text" placeholder="xxxxxxxxxx" [readonly]="shop.value === 'no'">
```
Comment (JimHolman1199) [inline src/app/component/ubs/components/ubs-order-details/ubs-order-details.component.html:169]: what is 'no' here ? if it some kind of response, please create enum for it

Comment (SofiiaBaziv) [reply src/app/component/ubs/components/ubs-order-details/ubs-order-details.component.html:169]: 'no' is a value of radio input. Do we need an enum for it?

**Review by VitalyBorysenko — APPROVED** [review]

File: src/app/component/ubs/components/ubs-order-details/ubs-order-details.component.ts (hunk 1 above)
Comment (SofiiaBaziv) [reply src/app/component/ubs/components/ubs-order-details/ubs-order-details.component.ts]: updated

File: src/app/component/ubs/components/ubs-order-details/ubs-order-details.component.html (hunk 2 above)
Comment (JimHolman1199) [reply src/app/component/ubs/components/ubs-order-details/ubs-order-details.component.html:169]: i think  we need to move to ts file
someCollName: object = { no: 'no', yes: 'yes' }

**Review by Edw1x — APPROVED** [review]

**Pull request closed without merging**